src/Testcontainers.Bigtable/BigtableBuilder.cs

```diff
@@ -0,0 +1,61 @@
+namespace Testcontainers.Bigtable;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public class BigtableBuilder: ContainerBuilder<BigtableBuilder, BigtableContainer, BigtableConfiguration>
+{
+  public const string GCloudCliDockerImage = "gcr.io/google.com/cloudsdktool/google-cloud-cli";
+
+  public const ushort BigtablePort = 9000;
+
+  public const string CMD = "gcloud beta emulators bigtable start --host-port 0.0.0.0:9000";
```

Comment:
Can we pass the command as an exec form to the builder configuration? Similar to [PubSub](https://github.com/testcontainers/testcontainers-dotnet/blob/2229e964cf0ed5b0b5daeac5d8eca5fafdb894fd/src/Testcontainers.PubSub/PubSubBuilder.cs#L46-L47)?

Reply:
Done. Didn't consider that this was an option.